checkPropTypes.js

```diff
@@ -9,13 +9,19 @@
 
 'use strict';
 
-var invariant = require('fbjs/lib/invariant');
+var emptyFunction = require('fbjs/lib/emptyFunction');
 var warning = require('fbjs/lib/warning');
 
 var ReactPropTypesSecret = require('./lib/ReactPropTypesSecret');
 
+var invariant = emptyFunction;
```

Comment:
Let's call this `invariantInDevOnly`. This is not how we usually use `invariant` so it might be very confusing.